vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Vagrant
+
+Here you will find a `Vagrantfile` to build a development environment for `kvm-vmi`.
+
+# Requirements
+
+- `vagrant`
+- `vagrant-libvirt` plugin
+- `ansible >= 2.2.1.0` (use a virtualenv, install it via pip)
```

Comment:
is the virtualenv necessary? if not, I'd live it out. If so, I'd show how to use it.